### ISSUE TITLE
A11y_SQLAzureDataStudioMAC_Import Extension - Modify Columns_VoiceOver: The table data and table column headers such as 'PassengerID', 'Name', 'Age' etcetera (depends on the file imported from system) are being read twice when moving through the table using Voice-over arrow keys.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/azuredatastudio#19883]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudioMAC_June2022;#SQL Azure Data Studio;#MAC;#FTP;#DesktopApp;#VoiceOver;#AINotInScope;#WCAG1.3.1;

## Environment Details:
Application Name: SQL Azure Data Studio
Application Version: Version 1.37.0
Mac Version: macOS Monterey 12.4
Screen Reader: Voice-Over

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch SQL Azure data studio.
3. Connect to a server.
4. Right click on the server and select 'Import Wizard'
5. Tab to the 'Preview Data' section.
6. Tab to 'ID' column header and table data.
7. Turn on Voice-over and observe.

## Actual:
The table data and table column headers such as 'PassengerID', 'Name', 'Age' etcetera (depends on the file imported from system) are being read twice when moving through the table using Voice-over arrow keys.

**The same issue can be observed throughout the application.**

## Expected:
The table data and table column headers such as 'PassengerID', 'Name', 'Age' etcetera should be read once when moving through the table using Voice-over arrow keys.

## User Impact:
Screen readers users will face inconvenience when the same information is announced twice continuously for all the elements inside a control.

## Attachment:
![24_Modify columns](https://user-images.githubusercontent.com/98393966/176434372-955b7c09-c2a7-431f-93a5-f4acbc28dbe8.gif)

**Comment by aasimkhan30** [comment]
@alanrenmsft , the voiceover is reading both the aria label and the contents of the cells. What will be the best way to deal with this. I was thinking of removing the aria-label from the cell.

**Comment by alanrenmsft** [comment]
how about on windows? @aasimkhan30

**Comment by aasimkhan30** [comment]
It does not make a difference in windows. Windows will announce the cell content only once regardless if aria-label is present or not.

**Comment by alanrenmsft** [comment]
@bbarik06 on Windows, the feature work as expected, the guideline is that we don't optimize for a specific screen reader. so we are not going to fix this. please close the issue.

**Comment by sspatil123** [comment]
GithubTags:#Closed;Version 1.41.2
![image](https://user-images.githubusercontent.com/85863999/220017941-b7752868-b398-45ae-ad2c-4c59de592c8e.png)
